src/composer.ts

```diff
@@ -8,6 +8,12 @@ import Context from './context'
 export type MaybeArray<T> = T | T[]
 export type MaybePromise<T> = T | Promise<T>
 export type NonemptyReadonlyArray<T> = readonly [T, ...T[]]
+export type DefinitelyDefined<T> = T extends infer S | undefined ? S : T
+export type Expand<T> = T extends object
```

Comment:
Could you share what this does?

Reply:
Expands the type from being

```TS
Omit<tg.CallbackQuery, 'data'> & {
      game_short_name: DefinitelyDefined<tg.CallbackQuery['game_short_name']>
}
```

To just the object of keys and values. Just nicer UX.

Reply:
I see, thanks